Makefile

```diff
@@ -159,7 +170,19 @@ build-ui: install-deps-ui  ## Builds the UI
 
 # Dockerized skywire-visor
 docker-image: ## Build docker image `skywire-runner`
-	docker image build --tag=skywire-runner --rm  - < skywire-runner.Dockerfile
+	#docker image build --tag=skywire-runner --rm  - < skywire-runner.Dockerfile
```

Comment:
Probably can be removed. Check if there are any references to the skywire-runner file and remove it if we can.

Reply:
I think it's not being used, but I'll investigate more. If it's not being used anywhere, I'll remove it on follow-up commits.